app/services/idv/proofer.rb

```diff
@@ -1,37 +1,21 @@
 module Idv
   module Proofer
-    @vendors = nil
-
     class << self
-      def validate_vendors!
```

Comment:
heads up this may conflict if my PR merges first: https://github.com/18F/identity-idp/pull/4464/files#diff-23e1ebd0b7b589abb3a8a3b5f9f0536eb8ab6c41499b1b0dfad67cc58d9100b3